views/js/qtiCreator/widgets/component/minMax/minMax.js

```diff
@@ -177,7 +180,7 @@ define([
                 if ( isFieldSupported(field) && _.isNumber(intValue) &&
                      intValue >= config.lowerThreshold && intValue <= config.upperThreshold ) {
 
-                    if ( this.is('rendered') && controls[field].input.val() !== intValue ) {
+                    if ( this.is('rendered') && parseInt(controls[field].input.val()) !== intValue ) {
```

Comment:
in case `allowDecimal: true` we need to do parseFloat?

Reply:
we can use `parseFloat` for both cases